### PR TITLE
A few improvements for Docker

### PR DESCRIPTION
**Pull request by jzila**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -41,7 +41,6 @@ after_failure:
     fi
 after_success:
   - if [ $TRAVIS_OS_NAME == 'linux' ] && [ $TRAVIS_BRANCH == 'master' ] && [ $TRAVIS_PULL_REQUEST == 'false' ]; then
-      docker login -e $CI_EMAIL -u $CI_USER -p $CI_PASS $CI_HOST;
```
Comment (cjb) [inline .travis.yml]: Why isn't this line needed anymore?

Comment (jzila) [inline .travis.yml]: it's already done above

**Comment by cjb** [comment]
@jzila oops, left some comments on the commits directly, not sure if you'll get email about those but you can see them here: https://github.com/keybase/client/pull/2272

**Comment by jzila** [comment]
Yep I got the email thanks.

**Comment by cjb** [comment]
ok, :thumbsup:

**Comment by jzila** [comment]
Closing this because it'll keep failing CI due to Docker requirement.

**Pull request closed without merging**